tools/gen_display_resources.py

```diff
@@ -61,7 +61,6 @@ def _load_row(self, y, row):
 # Get each glyph.
 for c in set(all_characters):
     if ord(c) not in f._glyphs:
-        print("Font missing character:", c, ord(c))
```

Comment:
Ditto about commenting out. Eventually we might want to find the missing characters?

Reply:
I am a firm believer in "delete, don't comment out", but if you feel strongly I'll do as you suggest.

Reply:
This is a simple thing to restore. I was thinking more about the computations in the (incorrect) "too long" message.